From 8ec6c677fcb034be3b84048e4db5f89a50424853 Mon Sep 17 00:00:00 2001
From: Hugo Osvaldo Barrera <hugo@whynothugo.nl>
Date: Mon, 5 Oct 2026 22:57:57 +0200
Subject: [PATCH] Fix build failure when using system nspr

Not suitable for upstream; assumes --system-nspr is always used.

See: https://phabricator.services.mozilla.com/D320363
---
 build.sh | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/build.sh b/build.sh
index 3eb34a2c1..0fa4a3eef 100755
--- a/build.sh
+++ b/build.sh
@@ -333,7 +333,7 @@ generate_pkg_config()
     [ -n "$vmajor" ] || return 0
 
     nspr_version=$(sed -n 's/^Version: *//p' \
-        "$obj_dir/lib/pkgconfig/nspr.pc" 2>/dev/null)
+        "/usr/lib/pkgconfig/nspr.pc" 2>/dev/null)
     : "${nspr_version:=4.32}"
 
     mkdir -p "$obj_dir/lib/pkgconfig"
-- 
2.55.0

